$NetBSD: patch-kodev_Makefile,v 1.6 2023/09/02 07:10:27 nia Exp $

Remove hardcoded optimizations.

-Werror makes it impossible to compile with OpenSSL 3.

--- kodev/Makefile.orig	2022-08-22 11:18:44.000000000 +0000
+++ kodev/Makefile
@@ -9,7 +9,7 @@ INSTALL_DIR=$(PREFIX)/bin
 
 S_SRC=	../src/cli.c
 
-CFLAGS+=-Wall -Werror -Wstrict-prototypes -Wmissing-prototypes
+CFLAGS+=-Wall -Wstrict-prototypes -Wmissing-prototypes
 CFLAGS+=-Wmissing-declarations -Wshadow -Wpointer-arith -Wcast-qual
 CFLAGS+=-Wsign-compare -Iincludes -std=c99 -pedantic
 CFLAGS+=-DPREFIX='"$(PREFIX)"'
@@ -17,8 +17,6 @@ LDFLAGS=-lcrypto
 
 ifneq ("$(NOOPT)", "")
 	CFLAGS+=-O0
-else
-	CFLAGS+=-O2
 endif
 
 ifneq ("$(MINIMAL)", "")
